.travis.yml

```diff
@@ -75,6 +76,12 @@ script:
       --spawn_strategy=sandboxed \
       --local_resources=400,2,1.0
   - |
+    if [[ "${TRAVIS_PYTHON_VERSION}" == 2* ]]; then
```

Comment:
Let's remove this from the build for now, as it doubles the build time. (We can remove the `pip install virtualenv` above, too, then.)